### PR TITLE
Edit documentation to point to the official docs repo

### PR DESCRIPTION
**Pull request by theletterf**
This edits the README and in-repo docs to point to the official documentation. The `CONTRIBUTING.md` file is also edited to comply with the GDI specification.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ See [SECURITY.md](SECURITY.md#reporting-security-issues) for instructions.
...
+The Splunk Observability documentation is hosted on https://docs.splunk.com/Observability,
+which contains all the prescriptive guidance for Splunk Observability products. 
 Prescriptive guidance consists of step-by-step instructions, conceptual material,
-and decision support for customers. Reference documentation and development
+and decision support for customers. Reference documentation and development 
 documentation is hosted on this repository.
 
-You can send feedback about Splunk Observability docs by clicking the Feedback
-button on any of our documentation pages.
+To contribute documentation for this project, open a pull request in the
```
Comment (pjanotti) [inline CONTRIBUTING.md:42]: "this project" read like this repo to me and got me confused for a bit, perhaps be explicit to get clarity at first reading, something like
```suggestion
To contribute to the Splunk Observability documentation, open a pull request in the 
```

File: README.md [hunk 2]
```diff
@@ -14,34 +14,24 @@ provides a .NET Profiler and startup hooks
...
-[Troubleshooting](docs/troubleshooting.md)
-documentation.
+For instructions on how to migrate from the SignalFx Instrumentation for .NET to the Splunk Distribution of OpenTelemetry .NET, see [Migrate from the SignalFx .NET Instrumentation](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=otel.net.migrate) in the official documentation.
+
+## Troubleshooting
 
-<!-- To be used when we have our official docs:
 For troubleshooting information, see the
-[Troubleshooting](https://docs.splunk.com/Observability/gdi/get-data-in/application/dotnet/troubleshooting/common-dotnet-troubleshooting.html)
+[Troubleshooting](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=otel.net.tshoot)
```
[the remark this answers is not in the document]
Comment (theletterf) [reply README.md:33]: We're using Quickdraw links to facilitate CTR tracking and updates to links should URLs change. Tech writers have full control of those.

File: docs/README.md [hunk 3]
```diff
@@ -1,111 +1,3 @@
...
-
-If you're currently using the [SignalFx Instrumentation for .NET](https://github.com/signalfx/signalfx-dotnet-tracing)
-and want to migrate to the Splunk Distribution of OpenTelemetry .NET,
-see the [MIGRATING](../MIGRATING.md) documentation.
-
-## Troubleshooting
-
-For troubleshooting information, see the
-[Troubleshooting](troubleshooting.md) documentation.
+## [Instrument NodeJS applications for Splunk Observability Cloud](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=otel.net.application)
```
[the remark this answers is not in the document]
Comment (theletterf) [reply docs/README.md:3]: Thanks for spotting the copypasta!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pjanotti) [reply README.md:33]: Do we have a list somewhere?

**Review by pjanotti — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (theletterf) [reply README.md:33]: Yes! Sending it to you via DM.

**Review by Kielek — APPROVED** [review]
Please fix markdown lint issues.

**Comment by theletterf** [comment]
@Kielek Done! This linter is _so_ picky! :D

**Pull request merged**